.flutter-plugins

```diff
@@ -1,11 +1,12 @@
 # This is a generated file; do not edit or check into version control.
-camera=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/camera-0.7.0+4/
-mic_stream=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/mic_stream-0.4.0/
-path_provider=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/path_provider-2.0.1/
-path_provider_linux=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/path_provider_linux-2.0.0/
-path_provider_macos=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/path_provider_macos-2.0.0/
-path_provider_windows=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/path_provider_windows-2.0.1/
-permission_handler=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/permission_handler-5.1.0+2/
-qr_code_scanner=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/qr_code_scanner-0.4.0/
-runevm_fl=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/runevm_fl-0.1.4/
-sensors=/Users/geerttrooskens/.pub-cache/hosted/pub.dartlang.org/sensors-0.4.2+6/
+camera=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.7.0+4/
+import_js_library=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/import_js_library-1.0.2/
+mic_stream=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/mic_stream-0.4.0/
+path_provider=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-2.0.2/
+path_provider_linux=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_linux-2.0.0/
+path_provider_macos=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_macos-2.0.0/
+path_provider_windows=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_windows-2.0.1/
+permission_handler=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/permission_handler-5.1.0+2/
+qr_code_scanner=/usr/local/flutter/.pub-cache/hosted/pub.dartlang.org/qr_code_scanner-0.4.0/
+runevm_fl=/Users/kthakore/Documents/HOTG-ai/runevm_fl/
```

Comment:
These files shouldn't be committed to this in the first place i think? Update gitignore?